src/System.Text.Json/tests/Serialization/DictionaryTests.KeyPolicy.cs

```diff
@@ -97,30 +122,18 @@ public static void NullNamePolicy()
                 DictionaryKeyPolicy = new NullNamingPolicy()
             };
 
-            // A policy that returns null is not allowed.
-            Assert.Throws<InvalidOperationException>(() => JsonSerializer.Deserialize<Dictionary<string, int>>(@"{""onlyKey"": 1}", options));
+            // A naming policy that returns null is not allowed.
             Assert.Throws<InvalidOperationException>(() => JsonSerializer.Serialize(new Dictionary<string, int> { { "onlyKey", 1 } }, options));
-        }
-
-        [Fact]
-        public static void KeyConflictDeserialize_LastValueWins()
```

Comment:
Isn't this still a useful test case? What do we do for this case now since we are no longer using naming policy when deserializing? The behavior should be captured in a test.